related-tooling.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Related Tooling
+sidebar: home_sidebar
+permalink: /related-tooling/
+toc: false
+---
+
+Open Source tooling, such as for managing the state of Pulp and integrating it with other projects, that is either maintained by the commnunity or other projects, or that is developed by the Pulp Project but not yet in production.
+
+## Foreman / Katello
+
+[Foreman](https://theforeman.org/) is a complete lifecycle management tool for physical and virtual servers. [Katello](https://theforeman.org/plugins/katello/) brings the full power of content management alongside the provisioning and configuration capabilities of Foreman.
+
+Foreman & Katello use Pulp for managing repositories software packages. It provides a graphical WebUI for Pulp and integrates it with its workflows, but does not expose every underlying feature & content type of Pulp.
```

Comment:
"managing repositories(['] or [of] added here) software packages"

Also, there is mixed use of "and" and "&" throughout, perhaps pick one and stay consistent?

Reply:
I'll fix that "managing repositories software packages" typo.

I wrote the 2nd paragraph, but copied & pasted the 1st.

I'll replace "&" with "and".

As my own personal style guide, I like to use "&" for words that commonly go together ("peanut butter & jelly"), but use "and" for everything else ("I went to the store and bought tea.") However, this page is not my own personal document.